todo-backend/README.md

```diff
@@ -0,0 +1,60 @@
+# TODO Backend
+
+This actor implements the [Todo backend spec](https://github.com/TodoBackend/todo-backend-js-spec/blob/master/js/specs.js).
+
+This actor makes use of the HTTP server (`wasmcloud:httpserver`) capability, the key-value store capability (`wasmcloud:keyvalue`) and the logging capability (`wasmcloud:logging`). As usual, it is worth noting that this actor does _not_ know where its HTTP server comes from, nor does it know which key-value implementation the host runtime has provided.
+
+## How to run
+
+- To start off you will need redis:
+
+```bash
+docker run -p 6379:6379 --name todo-backend-store -d redis
+```
+
+- Run `make` and `wash claims inspect ./target/wasm32-unknown-unknown/debug/todo_backend_s.wasm --output json | jq .module -r` from which you can extract the `<Actor id>`.
+
+- Run `export TODO_ACTOR=<Actor id (called Module in the above output)>`.
+
+- Run `RUST_LOG=info wasmcloud -m manifest.yaml`. This will trace all `info` type logs from all the capability providers including `wasmcloud:logging`.
```

Comment:
I'd love to see a `make run` target here that launches redis and exports the TODO_ACTOR environment variable for you, just to make the example super-easy to run. I did a similar target for another example if you need inspiration (especially because Makefiles are notoriously annoying with environment variables) https://github.com/wasmCloud/examples/pull/49/files#diff-417fbc3bca03a4f7b3dc86007b4b30731abc019d2344be320059565d23799cb1R23-R25